content/hi/docs/concepts/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: अवधारणाएँ
+main_menu: true
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है जिसका उपयोग कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए करता है, और कुबेरनेट्स कैसे काम करता है, इसकी गहरी समझ प्राप्त करने में आपकी मदद करता है।
```

Comment:
```suggestion
अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है जिसका उपयोग कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए करता है, और कुबेरनेट्स कार्यप्रणाली की गहरी समझ प्राप्त करने में आपकी मदद करता है।
```

Reply:
Sounds better to me, wdyt?

Reply:
This sounds much better and captures the meaning of what the docs wanted to convey.